tests/integration/conftest.py

```diff
@@ -180,6 +192,14 @@ def pytest_addoption(parser):
         type=str,
         help="run tests using a specific WPK package version"
     )
+    parser.addoption(
+        "--add-file",
+        action="append",
+        metavar="add-file",
```

Comment:
```suggestion
        metavar="log",
```

Reply:
I think that name should be clear and short. 
I suggest `log` but it only an idea.

Reply:
maybe log-file would be a more suitable name, since it adds a file that would serve as part of the log, but is not necesarily a .log per se (we could pull a conf, json, txt, etc).

```suggestion
        metavar="log-file",
```